bigquery_etl/dryrun.py

```diff
@@ -577,7 +577,7 @@ def validate_schema(self):
             project_name, dataset_name, table_name, partitioned_by
         )
 
-        if not query_schema.compatible(table_schema):
+        if not table_schema.compatible(query_schema):
```

Comment:
I think the way it was before is correct: 


https://github.com/mozilla/bigquery-etl/blob/681ac4b73abf88b7836009336e968294c0433fe2/bigquery_etl/schema/__init__.py#L123-L126 

The description is slightly confusing perhaps, but the `compatible` method essentially tests for two cases in the old version:
1) assuming `query_schema` has two fields `a` and `b` but the `table_schema` has 3 fields `a`, `b` and `c` then the schemas would still be compatible, because writing the query results to the table would just result in `c` being NULL
2) assuming assuming `query_schema` has 3 fields `a`, `b` and `c`  but the `table_schema` has two fields `a` and `b` then the schemas are not compatible since the values of `c` cannot get written to the table

If the schemas get flipped then the checks won't work correctly anymore